code/obj/item/gun/kinetic.dm

```diff
@@ -1165,6 +1213,30 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 						C.forensic_ID = src.forensic_ID
 						C.set_loc(T)
 
+/obj/item/gun/kinetic/single_action/mts_255
+	name = "\improper MTs-255 Revolver Shotgun"
+	desc = "A single-action revolving cylinder shotgun, popular with Soviet hunters, produced by the Zvezda Design Bureau."
+	icon = 'icons/obj/large/48x32.dmi'
+	icon_state = "mts255"
+	item_state = "mts255"
+	force = MELEE_DMG_RIFLE
+	contraband = 5
+	ammo_cats = list(AMMO_SHOTGUN_ALL)
+	max_ammo_capacity = 5
+	auto_eject = FALSE
+	can_dual_wield = FALSE
+	two_handed = TRUE
+	has_empty_state = FALSE
+	has_uncocked_state = TRUE
+	fire_animation = TRUE
+	gildable = TRUE
+	default_magazine = /obj/item/ammo/bullets/a12/weak/five
```

Comment:
Want to highlight this since it is a very big change, if this becomes the replacement for safe shotguns. But as it currently stands, it's my opinion that giving a person rubber slugs in a safe makes zero sense.

If they're using it for the intended purpose of fighting in an azone, debris field, etc. Rubber slugs near useless against the critters they need to fight. The only reason a person would likely pick up the current shotgun, is to:
1. Fire rubber slugs at people back on station for non-lethal fun
2. Swap it out with homemade shells for lethals

Reply:
From feedback, switched to scrapshot.